### PR TITLE
Update Zendesk credentials documentation

### PR DESCRIPTION
**Pull request by harshil1712**

### CODE REVIEW

File: docs/nodes/credentials/Zendesk/README.md [hunk 1]
```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
 
 ## Using OAuth
 
-::: tip ⛅️ Note for n8n.cloud users
-You'll only need to enter the Credentials Name and click on the circle button in the OAuth section to connect your Zendesk account to n8n.
-:::
-
 1. Access your Zendesk dashboard.
```
Comment (tanay1337) [inline docs/nodes/credentials/Zendesk/README.md:18]: Is there a generic link to the dashboard that we can share?

File: docs/nodes/credentials/Zendesk/README.md [hunk 2]
```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
...
-4. Click on the 'OAuth Clients' tab.
-5. Enter the client name in the ***Client Name*** field.
-6. Enter the description in the ***Description*** field.
-7. Copy the 'OAuth Callback URL' provided in the 'Zendesk OAuth2 API' credentials in n8n and paste it in the ***Redirect URLs*** field in the Zendesk API credentials page.
-8. Click on the ***Save*** button.
-9. Use the displayed ***Secret*** in the 'Zendesk OAuth2 API' credentials in n8n.
-10. Enter the client name you used in the Zendesk API credentials page in the ***Client ID*** field in n8n.
-11. Click on the circle button in the OAuth section to connect a Zendesk account to n8n.
-12. Click on the ***Save*** button to save your credentials.
+3. Click on 'API' under the ***Channels*** section in the sidebar.
```
Comment (tanay1337) [inline docs/nodes/credentials/Zendesk/README.md:20]: CHANNELS

File: docs/nodes/credentials/Zendesk/README.md [hunk 3]
```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
...
+4. Click on the ***OAuth Clients*** tab.
+5. Click on the ***Add OAuth client*** button.
+6. Enter the client name in the ***Client Name*** field.
+7. Enter a description in the ***Description*** field.
+8. Copy the 'OAuth Callback URL' provided in the 'Zendesk OAuth2 API' credentials in n8n and paste it in the ***Redirect URLs*** field on the Zendesk API credentials page.
```
Comment (tanay1337) [inline docs/nodes/credentials/Zendesk/README.md:25]: Split into two?

File: docs/nodes/credentials/Zendesk/README.md [hunk 4]
```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
...
+9. Paste it in the ***Redirect URLs*** field.
```
Comment (tanay1337) [inline docs/nodes/credentials/Zendesk/README.md:26]: Delete this

File: docs/nodes/credentials/Zendesk/README.md [hunk 5]
```diff
@@ -15,31 +15,54 @@ Create a [Zendesk](https://zendesk.com/) account.
...
+10. Click on the ***Save*** button.
+11. Click on the ***OK*** button on the ***Please store the secret that will appear*** pop-up.
+12. Copy the displayed ***Secret***.
```
Comment (tanay1337) [inline docs/nodes/credentials/Zendesk/README.md:29]: Scroll down and ...

**Review by tanay1337 — CHANGES_REQUESTED** [review]

File: docs/nodes/credentials/Zendesk/README.md (hunk 1 above)
Comment (harshil1712) [reply docs/nodes/credentials/Zendesk/README.md:18]: Zendesk doesn't provide a generic link.

**Pull request merged**